### PR TITLE
Add merqury last version

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/merqury/merqury.xml [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+            </output_collection>
+            <output_collection name="stats_files" type="list" count="1">
+                <element name="output_02.completeness" file="output_02.completeness.stats.tabular" ftype="tabular"/>
+            </output_collection>
+            <output_collection name="qv_files" type="list" count="2">
+                <element name="output_02.assembly"/>
+                <element name="output_02" file="output_02_qv.tabular" ftype="tabular"/>
+            </output_collection>
+            <output_collection name="wig_files" type="list" count="1">
+                <element name="assembly_only" file="output_02.assembly_only.wig" ftype="wig"/>
```
Comment (bgruening) [inline tools/merqury/merqury.xml:259]: remove the file, its too big and include assert_contents

File: tools/merqury/merqury.xml [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+            </output_collection>
+            <output_collection name="bed_files" type="list" count="1">
+                <element name="assembly_only" file="output_02.assembly_only.bed" ftype="bed"/>
```
Comment (bgruening) [inline tools/merqury/merqury.xml:262]: remove the file, its too big and include assert_contents

**Comment by bgruening** [comment]
I think this one is good to go, if those two test-files are removed/replaced.

File: tools/merqury/merqury.xml (hunk 2 above)
Comment (gallardoalba) [reply tools/merqury/merqury.xml]: Done :)

**Comment by gallardoalba** [comment]
> I think this one is good to go, if those two test-files are removed/replaced.

Done! :)

File: tools/merqury/merqury.xml [hunk 3]
```diff
@@ -256,10 +256,10 @@
                 <element name="output_02" file="output_02_qv.tabular" ftype="tabular"/>
             </output_collection>
             <output_collection name="wig_files" type="list" count="1">
-                <element name="assembly_only" file="output_02.assembly_only.wig" ftype="wig"/>
+                <element name="assembly_only"/>
             </output_collection>
             <output_collection name="bed_files" type="list" count="1">
-                <element name="assembly_only" file="output_02.assembly_only.bed" ftype="bed"/>
+                <element name="assembly_only"/>
```
[the remark this answers is not in the document]
Comment (gallardoalba) [reply tools/merqury/merqury.xml]: Sure.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks a lot @gallardoalba !

**Pull request merged**